pie/storage/database.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from typing import Dict, Optional, Union
+
+from sqlalchemy import BigInteger, Column, String
+
+from pie.database import database, session
+
+
+class Storage(database.base):
+    __tablename__ = "storage_data"
+
+    module = Column(String, primary_key=True)
+    guild_id = Column(BigInteger, primary_key=True)
+    key = Column(String, primary_key=True)
+    value = Column(String)
+
+    @staticmethod
+    def set(
+        module: str,
+        key: str,
+        value: str,
+        guild_id: int = 0,
+        ignore_existing: bool = False,
```

Comment:
If this is needed, I'd name it `allow_overwrite`... It feels more descriptive. But I can be convinced otherwise.

Reply:
You are right, it's more accurate. Fixed.